protos/fpc/fpc.proto

```diff
@@ -78,7 +90,7 @@ message ChaincodeRequest {
     bytes return_encryption_key = 3;
 }
 
-message EncryptedChaincodeRequest {
+message EncryptedChaincodeRequestMessage {
```

Comment:
I think we can drop the `Encrypted` Prefix for ChaincodeRequest and Response Messages. For the request, the only field it contains already suggests that it is encrypted.

For the `ChaincodeResponseMessage` some fields, such as the signature, enclave_id, ... are in clear and only the response is encrypted.

I could imaging that if we wanna support cleartext requests and responses, we may use a simple one_of for the request.

Reply:
i see your point and also had similar feelings but when dropping the `Encrypted` it differs from the cleartext only by `Message` which is also not nice. I solved that now by making the inner one explicitly a Cleartext Request ..